source/reference/template-apps.txt

```diff
@@ -204,6 +204,9 @@ the Atlas App Services UI, Realm CLI, or GitHub.
 Template Apps Available
 -----------------------
 
+The following templates are available. The Realm CLI accepts the following IDs
```

Comment:
It's not obvious from this sentence in which commands I can use the `--template` flag, and when I tried using the search widget to find it, I did not get good results. Should we link directly to the docs for [realm-cli create](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/template-app-fixes/cli/realm-cli-apps-create/) and [realm-cli pull](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/template-app-fixes/cli/realm-cli-pull/)?